core/am/rest-cxf/src/main/java/org/apache/syncope/core/rest/cxf/service/wa/GoogleMfaAuthAccountServiceImpl.java

```diff
@@ -70,15 +70,25 @@ public void update(final GoogleMfaAuthAccount acct) {
     }
 
     @Override
-    public GoogleMfaAuthAccount findAccountFor(final String owner) {
-        return logic.findAccountFor(owner);
+    public PagedResult<GoogleMfaAuthAccount> findAccountsFor(final String owner) {
```

Comment:
Why returning a `PagedResult` with page number statically set to `1`? Return `List<GoogleMfaAuthAccount>` instead.